src/main/java/redis/clients/jedis/util/Pool.java

```diff
@@ -45,6 +49,14 @@ public void initPool(final GenericObjectPoolConfig poolConfig, PooledObjectFacto
     this.internalPool = new GenericObjectPool<T>(factory, poolConfig);
   }
 
+  public void prepareInternalPool() {
+    try {
+      this.internalPool.preparePool();
+    } catch (Exception e) {
+      logger.warn("failed to initialize minIdle connections in the internal jedis connection pool", e);
```

Comment:
Please change:
- Capitalize beginning of sentence.
- Remove `jedis`. This is a generic class (`Pool<T>`) and T may or may not be Jedis.

```suggestion
      logger.warn("Could not initialize minimum number of idle resources", e);
```